libs/shared/ui-components/src/SocialCard/types.ts

```diff
@@ -0,0 +1,33 @@
+// og:url -  The canonical URL of your object that will be used as its permanent ID in the graph,
+// example: https://labs.quansight.org/projects
+// example: https://quansight.com/
+// note: should it include query string params?
+// like https://quansight.com/library?page=2&type=blog (?)
```

Comment:
I did not intend you to copy these two lines verbatim 😊, sorry for the confusion 

These are questions. I'm not sure what the answer is. But whatever the answer is, it should be documented.

Reply:
Thanks for pointing this out, @gabalafou -- I believe it should *not* include query parameters.

The reason I think this is, the main way to bust Twitter's cache for the preview card for a link is to add spurious, new query parameters to that link when composing a test tweet.

If the query parameters mattered to the preview, then this would not be the recommended or observed behavior.

Reply:
Per [here](https://www.everywheremarketer.com/blog/ultimate-guide-to-social-meta-tags-open-graph-and-twitter-cards), if the query parameters can affect which page is shown, you should *include* them.

So, since the query parameters for `/library` on Consulting and `/blog` *do* affect the contents of the page (even if not the social card metadata), it will likely be safer to *include* the query params in `url`.